index.js

```diff
@@ -0,0 +1,91 @@
+function gameResult() {
```

Comment:
I'm kind of confused by this project. gameResult() only runs once - ever. So how does a user interact with it? Seems like if this was a live site, someone would have to regularly push the new HTML file for the JS updates to be made, but if so, why not hard-code the JS changes into the HTML file?